routes/api_v1_0.py

```diff
@@ -915,6 +921,382 @@ def mbt(projection: str, tiletype: str, zoom: int, x: int, y: int):
     return send_file(requestf)
 
 
+@bp_v1_0.route('/api/v1.0/observation/datatypes.json')
+def observation_datatypes_v1_0():
+    """
+    API Format: /api/v1.0/observation/datatypes.json
+
+    Returns the list of observational data types
+
+    **Used in ObservationSelector**
+    """
+    max_age = 86400
+    data = [
+        {
+            'id': dt.key,
+            'value': dt.name,
+        }
+        for dt in ob_queries.get_datatypes(DB.session)
+    ]
+    resp = jsonify(data)
+    resp.cache_control.max_age = max_age
+    return resp
+
+@bp_v1_0.route('/api/v1.0/observation/meta_keys/<string:platform_types>.json')
+def observation_keys_v1_0(platform_types: str):
+    """
+    API Format: /api/v1.0/observation/meta_keys/<string:platform_types>.json
+
+    <string:platform_types> : Comma seperated list of platform types
+
+    Gets the set of metadata keys for a list of platform types
+
+    **Used in ObservationSelector**
+    """
+    max_age = 86400
+    data = ob_queries.get_meta_keys(DB.session, platform_types.split(','))
+    resp = jsonify(data)
+    resp.cache_control.max_age = max_age
+    return resp
+
+@bp_v1_0.route('/api/v1.0/observation/meta_values/<string:platform_types>/<string:key>.json')
+def observation_values_v1_0(platform_types: str, key: str):
+    """
+    API Format: /api/v1.0/observation/meta_values/<string:platform_types>.json
+
+    <string:platform_types> : Comma seperated list of platform types
+    <string:key> : Metadata key
+
+    Gets the set of metadata values for a list of platform types and key
+
+    **Used in ObservationSelector**
+    """
+    max_age = 86400
+    data = ob_queries.get_meta_values(
+        DB.session, platform_types.split(','), key
+    )
+    resp = jsonify(data)
+    resp.cache_control.max_age = max_age
+    return resp
+
+@bp_v1_0.route('/api/v1.0/observation/tracktime/<string:platform_id>.json')
```

Comment:
Could we increase specificity by using `tracktimerange` or something similar? Looking to indicate from this declaration that the URL handles a time range.

Reply:
Done.